manager/src/agent.rs

```diff
@@ -101,13 +126,15 @@ impl Contract {
         self.storage_unregister(Some(true));
     }
 
-    /// Allows an agent to withdraw all rewards, paid to the specified payable account id.
-    ///
-    /// ```bash
-    /// near call cron.testnet withdraw_task_balance --accountId YOUR_AGENT.testnet
-    /// ```
-    pub fn withdraw_task_balance(&mut self) -> Promise {
-        let account = env::predecessor_account_id();
+    /// Removes the agent from the active set of agents.
+    /// Withdraws all reward balances to the agent payable account id.
+    #[private]
+    pub fn exit_agent(&mut self, account_id: Option<AccountId>, remove: Option<bool>) -> Promise {
+        let mut account = env::predecessor_account_id();
+        if let Some(account_id) = account_id {
+            account = account_id;
+        }
```

Comment:
I've seen Austin do this:
`let account = account_id.unwrap_or_else(env::predecessor_account_id);`
https://github.com/near/near-sdk-rs/blame/95954cad746502aeb64e42a8bd5bbbb1ce0df8cc/near-contract-standards/src/fungible_token/storage_impl.rs#L51

Reply:
hahahahaahaha... ha... ah. facepalm easy